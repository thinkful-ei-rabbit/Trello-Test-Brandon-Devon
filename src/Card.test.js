import React from 'react';
import ReactDOM from 'react-dom';
import renderer from 'react-test-renderer';
import Card from './Card';

//smoke test:
it('renders without crashing', () => {
    const div = document.createElement('div');
    ReactDOM.render(<Card 
        title="Card one"
        content="card content"
        />, 
        div
    );
    
    ReactDOM.unmountComponentAtNode(div)
});

//snapshot test:
it('renders the UI as expeted', () => {
    const tree = renderer.create(<Card  />).toJSON();
    expect(tree).toMatchSnapshot();
});

it('renders the UI as expeted with no card', () => {
    const tree = renderer.create(<Card  />).toJSON();
    expect(tree).toMatchSnapshot();
});