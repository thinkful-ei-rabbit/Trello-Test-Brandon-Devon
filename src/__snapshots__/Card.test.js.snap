// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders the UI as expeted 1`] = `
<div
  className="Card"
>
  <button
    type="button"
  >
    delete
  </button>
  <h3 />
  <p />
</div>
`;

exports[`renders the UI as expeted with no card 1`] = `
<div
  className="Card"
>
  <button
    type="button"
  >
    delete
  </button>
  <h3 />
  <p />
</div>
`;
